src/components/Correction/AgreementType.vue

```diff
@@ -93,6 +93,20 @@ export default class AgreementType extends Vue {
   private agreementType: string = null
   private showAgreementTypeForm = false
 
+  get hasAgreementTypeChange (): boolean {
+    return (
+      this.agreementType !== this.getOriginalIA?.incorporationApplication?.incorporationAgreement.agreementType
+    )
+  }
+
+  get selectedAgreementDescription () : string {
+    if (this.getAgreementType) {
+      return AgreementTypeResource.find(item => item.code === this.getAgreementType)
+        .summaryDescription
+    }
+    return ''
+  }
+
```

Comment:
Moved getters up from below.

Also added conditional chaining operator here since getters fire before the original IA is fetched (and I now initialize it to null).